lib/ReactViews/Workbench/Controls/dimension-selector-section.scss

```diff
@@ -1,12 +1,46 @@
 @import "~terriajs-variables";
 
 .dimension-selector {
-  font-size: $font-size-mid-small;
+  font-size: 0.9rem;
+  margin-top: 10px;
   label {
     margin-right: $padding-small;
   }
 }
 
+.dimension-selector-select {
+  position: relative;
+  svg {
+    position: absolute;
+    fill: $text-light;
+    width: 10px;
+    top: 0;
+    height: 40px;
+    right: 10px;
+    pointer-events: none;
+  }
+  select {
+    background: rgba(250, 250, 250, 0.2);
```

Comment:
Will wait for your styled-components overhaul of this, but paying special attention to this to use theme colours as this will be the result if applied to another map like nsw in this form:
<img width="263" alt="image" src="https://user-images.githubusercontent.com/9134432/81256132-f176fd00-9072-11ea-9105-0a83afa77eda.png">

Reply:
And doubly acknowledging that it also isn't perfect in `mobx` in its un-merged StyleSelector/DimensionsSelector form